### DISCUSSION TITLE
Dynamically inserted date created and modified?

### DISCUSSION THREAD

**Discussion by mshostack** [akosbalasko/yarle#466] [Q&A] [answered]
I'm trying to migrate from Evernote to Obsidian. The default template in Yarle uses the following frontmatter yaml:

```
{created-at-block}_Created at {created-at}._{end-created-at-block}
{updated-at-block}_Last updated at {updated-at}._{end-updated-at-block}
```

This seems to insert the actual date and time metadata from Evernote, but I dislike the template format.

1. Can I just use the `{created-at}` and `{updated-at}` to insert it without the surrounding blocks? I'm not even sure what the blocks do.
2. When I try to add those above dynamic macros in a Core template in Obsidian, they don't work. Is this something specific to Yarle? How can I add those dynamic values for all notes in Obsidian without overcomplicating things with Templater?

**Comment by akosbalasko** [comment, marked as answer]
Hi @mshostack ,
In the UI, the template is editable, so feel free to change it as you wish there, just remove the" _Created at" and "_Last updated_at" texts, and the "._"-s from the end. The other ({created-at-block}, {end-created-at-block}  and the other similars in the next line) are mandatory for Yarle to identify the different sections, but they will be erased after the conversion, so they should not appear in the produced markdown file. 
If they appear, please send me the config settings (at the beginning in the logs), because that whould be a misconfiguration or a bug. Thank you! 

Hope it helps,
akos
